ros_randla_net:
  topics:
    pcl_sub: /pointcloud/orig
    pcl_pub: /pointcloud/seg